src/components/PictureGallery/PictureGallery.js

```diff
@@ -41,6 +71,32 @@ function PictureGallery({ protestId, date }) {
   );
 }
 
+function ImageDateSection({ src, date, onClick }) {
+  return (
+    <ImageDateContainer onClick={onClick}>
+      <ImageThumbnail src={src} />
+      <DateTitle>{dateToDayOfWeekAndDate(date)}</DateTitle>
+    </ImageDateContainer>
+  );
+}
+
+function getPictureForEveryDate(pictures) {
```

Comment:
Can you move these helper functions to `PictureGalleryUtils.js`, under the `PictureGallery` component folder, and add a JSDoc for each of them?